Thu Oct 30 15:42:01 CST 2025 : Apache Derby 网络服务器 - 10.14.2.0 - (1828579) 已启动并准备接受端口 1527 上的连接
----------------------------------------------------------------
Thu Oct 30 15:42:01 CST 2025:
正在使用类加载程序 jdk.internal.loader.ClassLoaders$AppClassLoader@6bc7c054 引导数据库
目录 C:\Users\Lenovo\Desktop\P18_25294349_25294047\Blackjack\BlackjackDB\BJ 中的 Derby 版本 The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579) 实例 a816c00e-019a-3411-03ff-00002974f5c4
已从 file:/C:/Users/Lenovo/Desktop/db-derby-10.14.2.0-bin/db-derby-10.14.2.0-bin/lib/derby.jar 加载
java.vendor=Oracle Corporation
java.runtime.version=23.0.2+7-58
user.dir=C:\Users\Lenovo\Desktop\db-derby-10.14.2.0-bin\db-derby-10.14.2.0-bin
os.name=Windows 11
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Lenovo\Desktop\P18_25294349_25294047\Blackjack\BlackjackDB
数据库类加载程序已启动 - derby.database.classpath=''
----------------------------------------------------------------
Thu Oct 30 15:42:02 CST 2025:
正在使用类加载程序 jdk.internal.loader.ClassLoaders$AppClassLoader@6bc7c054 关闭数据库目录 C:\Users\Lenovo\Desktop\P18_25294349_25294047\Blackjack\BlackjackDB\BJ 上的实例 a816c00e-019a-3411-03ff-00002974f5c4
Thu Oct 30 15:42:02 CST 2025 Thread[#35,DRDAConnThread_2,5,main] (DATABASE = BJ), (DRDAID = {2}), 数据库“BJ”已关闭。
----------------------------------------------------------------
Thu Oct 30 15:42:12 CST 2025:
正在使用类加载程序 jdk.internal.loader.ClassLoaders$AppClassLoader@6bc7c054 引导数据库
目录 C:\Users\Lenovo\Desktop\P18_25294349_25294047\Blackjack\BlackjackDB\BJ 中的 Derby 版本 The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579) 实例 a65c80ac-019a-3411-03ff-00002974f5c4
已从 file:/C:/Users/Lenovo/Desktop/db-derby-10.14.2.0-bin/db-derby-10.14.2.0-bin/lib/derby.jar 加载
java.vendor=Oracle Corporation
java.runtime.version=23.0.2+7-58
user.dir=C:\Users\Lenovo\Desktop\db-derby-10.14.2.0-bin\db-derby-10.14.2.0-bin
os.name=Windows 11
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Lenovo\Desktop\P18_25294349_25294047\Blackjack\BlackjackDB
数据库类加载程序已启动 - derby.database.classpath=''
----------------------------------------------------------------
Thu Oct 30 15:44:31 CST 2025:
正在使用类加载程序 jdk.internal.loader.ClassLoaders$AppClassLoader@6bc7c054 关闭数据库目录 C:\Users\Lenovo\Desktop\P18_25294349_25294047\Blackjack\BlackjackDB\BJ 上的实例 a65c80ac-019a-3411-03ff-00002974f5c4
Thu Oct 30 15:44:31 CST 2025 Thread[#53,DRDAConnThread_3,5,main] (DATABASE = BJ), (DRDAID = {5}), 数据库“BJ”已关闭。
----------------------------------------------------------------
Thu Oct 30 15:45:15 CST 2025:
正在使用类加载程序 jdk.internal.loader.ClassLoaders$AppClassLoader@6bc7c054 引导数据库
目录 C:\Users\Lenovo\Desktop\P18_25294349_25294047\Blackjack\BlackjackDB\BJ 中的 Derby 版本 The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579) 实例 522b00e8-019a-3411-03ff-00002974f5c4
已从 file:/C:/Users/Lenovo/Desktop/db-derby-10.14.2.0-bin/db-derby-10.14.2.0-bin/lib/derby.jar 加载
java.vendor=Oracle Corporation
java.runtime.version=23.0.2+7-58
user.dir=C:\Users\Lenovo\Desktop\db-derby-10.14.2.0-bin\db-derby-10.14.2.0-bin
os.name=Windows 11
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Lenovo\Desktop\P18_25294349_25294047\Blackjack\BlackjackDB
数据库类加载程序已启动 - derby.database.classpath=''
----------------------------------------------------------------
Thu Oct 30 15:45:15 CST 2025:
正在使用类加载程序 jdk.internal.loader.ClassLoaders$AppClassLoader@6bc7c054 关闭数据库目录 C:\Users\Lenovo\Desktop\P18_25294349_25294047\Blackjack\BlackjackDB\BJ 上的实例 522b00e8-019a-3411-03ff-00002974f5c4
Thu Oct 30 15:45:15 CST 2025 Thread[#53,DRDAConnThread_3,5,main] (DATABASE = BJ), (DRDAID = {7}), 数据库“BJ”已关闭。
Thu Oct 30 15:45:33 CST 2025 Thread[#53,DRDAConnThread_3,5,main] Cleanup action starting
java.sql.SQLException: 数据库“BJ”未找到。
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:571)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
Caused by: ERROR XJ004: 数据库“BJ”未找到。
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 21 more
============= begin nested exception, level (1) ===========
ERROR XJ004: 数据库“BJ”未找到。
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:571)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
============= end nested exception, level (1) ===========
Cleanup action completed
Thu Oct 30 15:45:33 CST 2025 Thread[#53,DRDAConnThread_3,5,main] (DATABASE = BJ), (DRDAID = {8}), 数据库“BJ”未找到。
----------------------------------------------------------------
Thu Oct 30 16:24:51 CST 2025：正在关闭 Derby 引擎
----------------------------------------------------------------
